legacy_plugins/insight_to_action/README.md

```diff
@@ -0,0 +1,457 @@
+# Insight To Action 
+> Pivot table with button for API calls
```

Comment:
@dremileta somehow we did not address this comment from https://gooddata.atlassian.net/browse/SPACE-2566 which is actually very important:

> Apart from that, i see the sdk-util and sdk-ui-pivot code from SDK.UI repo are copied here in their entirety. I’m sure this is intended. But we probably want to modify the READMEs, saying that these are copies of code from SDK.UI...

We have to mention here that the code comes from the SDK table, and that it is also a reason why this is a legacy plugin, that it's not a recommended way to copy the SDK component in order to modify them, but for some use cases, there is no other reasonable solution.